pkg/template/api/types.go

```diff
@@ -13,6 +13,9 @@ type Template struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Optional: An instructional message for the user processing this template (parameter substitution will be performed).
+	PostProcessMessage string
```

Comment:
let's standardize on this format for godoc'ing api fields:

// postProcessMessage is an optional instructional message for the user.......

i think that is the upstream intent although they are not following it religiously.